### PR TITLE
upload & insert file into database with node

### PR DESCRIPTION
**Pull request by sonirathore01**

### CODE REVIEW

File: demo4.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        img.img_path = request.body.path+request.body.file;
+        img.save(function (err,data) {
+            if(err) {
+                console.log(err);
+            }else{
+                console.log(data);
+            }
+        });
+
+
```
Comment (ghost) [inline demo4.js]: extra new lines

File: index4.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <title>Title</title>
+</head>
+<body>
+
+<form enctype="multipart/form-data" action="http://127.0.0.1:8086/upload" method="post">
+    <input type="file" name="photo" multiple />
+    <input type="submit" value="Upload Image" name="submit" />
+</form>
+
+
```
Comment (ghost) [inline index4.html:14]: extra new lines

File: demo4.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+/**
+ * Created by lcom73 on 24/1/17.
+ */
+var express = require('express');
+var multer = require('multer');
+var app = express();
+
+/* Disk Storage engine of multer gives you full control on storing files to disk. The options are destination (for determining which folder the file should be saved) and filename (name of the file inside the folder) */
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/mydb'); // connect to our database
```
Comment (ghost) [inline demo4.js:10]: connection string should be part of config file

Comment (sonirathore01) [reply demo4.js:10]: I'll take care next time..

**Pull request merged**